qiita_ware/ebi.py

```diff
@@ -620,6 +652,51 @@ def write_xml_file(self, element, fp):
         create_dir(self.xml_dir)
         ET.ElementTree(element).write(fp, encoding='UTF-8')
 
+    def generate_xml_files(self):
+        """Generate the XML files"""
```

Comment:
It is already a numpy doc. This function doesn't take parameters, doesn't return and doesn't raise anything specific.